### PR TITLE
upload new Matt Webb files

### PR DESCRIPTION
**Pull request by AreezG**
upload @mattdwebb 's new files, call table 6 in main

### CODE REVIEW

**Comment by AreezG** [comment]
@shichen14 when testing, please set the path in main.do and then run it :)

**Comment by shichen14** [comment]
Hi @AreezG , I have tested this and here are what I've noticed:

**Issue**: the 'main.do' was unable to complete executing on my computer.

**Reason**: backward slash ('\') used in do.file of Table 6 (line 8 and 69) and Table 7 (line 5) weren't interpreted as a path delimiter by macOS.

**Solution**: switch to forward slash ('/'). Stata will interpret them correctly as a path delimiter in all OS it supports.

(Tested in Stata 18.0 SE (rev. 26 Jul 2024) running on macOS 14.5)

File: table6.do [hunk 1]
```diff
@@ -0,0 +1,166 @@
+/* Stata Do File for Table 6 */
+/* Written by: Matthew D. Webb */
+/* Created: July 23, 2024 */
+
+clear
+
+// import data
+import delimited "${DATA}\HUDprocessed_JPE_census_042021.csv"
```
Comment (shichen14) [inline table6.do]: Backward slash used here will not be recognized by Stata running on macOS.

File: table6.do [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+
+foreach var in $VARS {
+	cap replace `var' = "." if `var' == ""
+	cap destring `var', replace force
+}
+
+
+/*corrected city*/
+	qui do "${CODE}\data_cleaner.do"
```
Comment (shichen14) [inline table6.do]: Backward slash used here will not be recognized by Stata running on macOS.

File: table7.do [hunk 3]
```diff
@@ -1,13 +1,8 @@
...
-/* Updated: September 4, 2023, by Anthony McCanny */
-/* updated: November 21, 2023, by Matt Webb */
-
+/*MDW - table 7*/
 clear
 
-// import data
-import delimited "${DATA}/HUDprocessed_JPE_census_042021.csv"
+
+import delimited "${DATA}\HUDprocessed_JPE_census_042021.csv"
```
Comment (shichen14) [inline table7.do]: Backward slash used here will not be recognized by Stata running on macOS.

**Review by shichen14 — COMMENTED** [review]
Main do file was unable to finish executing due to backward slash used in Table 6 and Table 7. Suggest switching to forward slash instead for better compatibility.

**Pull request merged**